packages/legend-application-studio/src/components/editor/editor-group/service-editor/testable/ServiceTestDataEditor.tsx

```diff
@@ -199,6 +201,12 @@ export const ConnectionTestDataEditor = observer(
       }
     };
 
+    const generateSkeletonTestData = (): void => {
```

Comment:
lets not sure skeleton in the code. 
we can say bare or just generate schemas

`generateSkeletonTestData ` -> `generateQuerySchemas`